examples/cloud_discovery_service/library_api/c++11/CloudDiscoveryServiceConfig.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/current/rti_cloud_discovery_service.xsd">
```

Comment:
```suggestion
      xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/7.0.0/rti_cloud_discovery_service.xsd">
```
We do a bulk change later on

Reply:
Do we typically do a bulk change of all the XSD versions with each release?

Reply:
Alright .. I will make the change.